Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -195,7 +195,14 @@
                 // everyone else are limited to within organizationnal context
                 itSystemBaseUrl = `/odata/Organizations(${this.user.currentOrganizationId})/ItSystems`;
             }
-            var itSystemUrl = itSystemBaseUrl + "?$expand=BusinessType($select=Name),BelongsTo($select=Name),TaskRefs($select=Description,TaskKey),Parent($select=Name,Disabled),Organization($select=Id,Name),Usages($select=OrganizationId;$expand=Organization($select=Id,Name)),LastChangedByUser($select=Name,LastName),Reference($select=Title,URL,ExternalReferenceId)";
+            var itSystemUrl = itSystemBaseUrl + "?$expand=BusinessType($select=Name)," +
+                                                "BelongsTo($select=Name)," +
+                                                "TaskRefs($select=Description,TaskKey)," +
+                                                "Parent($select=Name,Disabled)," +
+                                                "Organization($select=Id,Name)," +
+                                                "Usages($select=OrganizationId;$expand=Organization($select=Id,Name))," +
```

Comment:
Er det nødvendigt med OrganizationId? Der expandes jo på Organization hvori der selectes Id.

Reply:
det var for ikke at rette i den kode der bruger den (jeg har bare indsat linebreaks i dette PR), så jeg pillede ikke ved selve kendo koden, men jo det kan man jo godt sige er overflødigt